great_expectations/rule_based_profiler/expectation_configuration_builder/expectation_configuration_builder.py

```diff
@@ -30,6 +43,8 @@ def __init__(
 
         Args:
             expectation_type: the "expectation_type" argument of "ExpectationConfiguration" object to be emitted.
+            validation_parameter_builder_configs: ParameterBuilder configurations, having whose outputs available (as
```

Comment:
Not sure I understand this - these configs are optional right? And optionally instead of specifying them here they can be specified in the parameter_builder section of the config?

Reply:
@anthonyburdi Let's discuss -- I would like to offer a clarification, which I feel might be useful.  Thanks!

Reply:
Discussed clarifying this in a follow-on PR.